src/components/Conversation/Compose.vue

```diff
@@ -22,12 +22,13 @@
 <script>
 import ProfilePicture from '@/components/ProfilePictures/ProfilePicture'
 import { QItem, QItemMain, QInput, QBtn, QItemSide, QItemTile } from 'quasar'
+import statusMixin from '@/mixins/statusMixin'
 
 export default {
-  name: 'WallInput',
+  name: 'Compose',
```

Comment:
`ConversationCompose` might fit better, or could this also be a more standalone component?

Reply:
done